Added missed chats to doc (#572)
livechat/livechat-public-docs: content/data-reporting/reports-api/index.mdx

<Text>

> # Chat Missed

Reviewer: ```suggestion
# Chats Missed
```

Reviewer: why not Missed chats?

Author: My mistake with a copy...fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343

> This reports shows you the number of chats with no messages sent by agents (except the first welcome message that always occur in conversations).

Reviewer: ```suggestion
This report shows you the number of chats with no messages sent by agents (except the first welcome message that always occurs in conversations).
```

Author: fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343

## Optional arguments

| Argument | Description |
|---------|--------------------|
| `date_from` | `YYYY-MM-DD`, defaults to `the beginning of time` |
| `date_to` | `YYYY-MM-DD`, defaults to `today` |
| `timezone` | timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix) |

> | `groups[]` | id of the group, not set by default, returns statistics for the specified group |

Reviewer: ```suggestion
| `groups[]` | group ID, not set by default, returns statistics for the specified group |
```

Author: fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343

> | `agents[]` | agent's login, not set by default, return statistics for the specified agent |

Reviewer: ```suggestion
| `agents[]` | agent's login, not set by default, returns statistics for the specified agent |
```

Author: fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343

> | `group_by` | defaults to `day` (or `hour` when date_from equals date_to), can be set to `month`, `hour` or `day` |

Reviewer: ```suggestion
| `group_by` | defaults to `day` (or `hour` when `date_from` equals `date_to`), can be set to `month`, `hour` or `day` |
```

Author: fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343

> | `tag[]` | return statistics for the specified tag |

Reviewer: ```suggestion
| `tag[]` | returns statistics for the specified tag |
```

Author: fixed in https://github.com/livechat/livechat-public-docs/pull/572/commits/72db3c1bd3011dd7ce571afc3f608468d0a8b343
